ansible/roles/unmount-volume/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+# ansible-playbook playbooks/utils/unmount_volume.yml -i hosts --limit "vm0" -e "{ 'src':'/dev/vdc' }"
+
+
+- set_fact: ansible_python_interpreter=/usr/bin/python2.6
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version < '5')
+
+- name: register mountpoint and fstype
+  shell: grep "^/dev/vdc " /proc/mounts | cut -d ' ' -f 2,3 | awk '{ printf "[\"%s\",\"%s\"]\n", $1, $2 }' 
+  register: result
+
+- debug: Print out mountpoints for the device {{ src }}
+  msg="{{ (item |from_json )[0] }} {{ src }} {{ (item |from_json )[1] }}" 
+  with_items:
+    - "{{ result.stdout_lines }}"
```

Comment:
The `with_items` is to loop through all of the mountpoints that device has been attached to.
```
root@vm64-225:/# grep "^/dev/vdc " /proc/mounts | cut -d ' ' -f 2,3 | awk '{ printf "[\"%s\",\"%s\"]\n", $1, $2 }'                                                                 
["/vol2","ext3"]
["/vol1","ext3"]
```

And the `[0]` is to grab out the mountpoint data required. `[1]` is for fstype.